addons/web_editor/controllers/main.py

```diff
@@ -548,29 +548,46 @@ def subber(match):
             ('Cache-control', 'max-age=%s' % http.STATIC_CACHE_LONG),
         ])
 
-    @http.route(['/web_editor/media_library_url'], type='json', auth="user", website=True)
-    def media_library_url(self):
-        dbuuid = request.env['ir.config_parameter'].sudo().get_param('database.uuid')
+    @http.route(['/web_editor/media_library_search'], type='json', auth="user", website=True)
+    def media_library_search(self, **params):
         endpoint = request.env['ir.config_parameter'].sudo().get_param('web_editor.media_library_endpoint', DEFAULT_LIBRARY_ENDPOINT)
-        return '%s?dbuuid=%s' % (endpoint, dbuuid)
+        params['dbuuid'] = request.env['ir.config_parameter'].sudo().get_param('database.uuid')
+        response = requests.get('%s/1/search' % endpoint, params=url_encode(params))
```

Comment:
my bad, forgot to specify, will be better if it can be done in post, to avoid dbuid in log too